Apps/Src/main.c

```diff
@@ -98,7 +98,7 @@ void Task_Init(void *p_arg){
         (OS_MSG_QTY)0,
         (OS_TICK)0,
         (void*)NULL,
-        (OS_OPT)(OS_OPT_TASK_STK_CLR),
+        (OS_OPT)(OS_OPT_TASK_STK_CLR|OS_OPT_TASK_SAVE_FP|OS_OPT_TASK_SAVE_FP),
```

Comment:
Do the test files also need this change?

Reply:
All tasks that deal with floating point numbers do

Reply:
I still want to test this on hardware first too, just to be sure.